### PR TITLE
[microfrontends] allow non interactive creation of MFE group if within free tier billing

### PR DESCRIPTION
**Pull request by tim123abc**
Updates microfrontends create-group to support safe non-interactive usage on the free tier while still blocking non-interactive flows that would impact billing. 

Adds explicit non-interactive route flags for multi-project groups and expands unit tests to cover billing gates, flag validation, and route parsing edge cases.

### CODE REVIEW

File: packages/cli/src/commands/microfrontends/command.ts [hunk 1]
```diff
@@ -36,6 +36,14 @@ export const createGroupSubcommand = {
       deprecated: false,
       description: 'Default route for the default application',
     },
+    {
+      name: 'project-route',
```
Comment (mknichel) [inline packages/cli/src/commands/microfrontends/command.ts]: Should this be named `default-route` instead or `project-default-route`? At first I thought it would add paths to the microfrontends.json config.

File: packages/cli/src/commands/microfrontends/create-group.ts [hunk 2]
```diff
@@ -354,10 +405,9 @@ export default async function createGroup(client: Client): Promise<number> {
   }
   output.log('');
 
-  const confirmed = await client.input.confirm(
-    'Create microfrontends group?',
-    true
-  );
+  const confirmed = isNonInteractive
```
Comment (mknichel) [inline packages/cli/src/commands/microfrontends/create-group.ts]: should the agent have to specify `--yes` also?

File: packages/cli/src/commands/microfrontends/command.ts (hunk 1 above)
Comment (tim123abc) [reply packages/cli/src/commands/microfrontends/command.ts]: Fair point

File: packages/cli/src/commands/microfrontends/create-group.ts (hunk 2 above)
Comment (tim123abc) [reply packages/cli/src/commands/microfrontends/create-group.ts]: Hmm good point, going to make the approach similar to add to group etc

**Review by mknichel — APPROVED** [review]
